tox.ini

```diff
@@ -28,16 +37,13 @@ deps =
     django111: Django>=1.11,<2.0
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
-    # Django master builds are failing at the moment.
-    djangomaster: https://codeload.github.com/django/django/zip/master
```

Comment:
FTR we liked having the master runs in-case it was going to fail in the future version, but we wouldn't fail CI on it. You can do something similar with GitHub actions:

https://github.com/clokep/django-querysetsequence/blob/b5a6f06c4e83caeadcbd53816b55bc5cc0ecc03b/.github/workflows/main.yml#L40-L44